lib/stdlib/test/peer_SUITE.erl

```diff
@@ -563,3 +581,26 @@ docker(Config) when is_list(Config) ->
             ?assertEqual(Node, peer:call(Peer, erlang, node, [])),
             peer:stop(Peer)
     end.
+
+run_erl() ->
+    [{doc, "Test that it is possible to start a peer node using run_erl aka attached"}].
+
+run_erl(Config) ->
+    RunErl = os:find_executable("run_erl"),
+    [throw({skip, "Could not find run_erl"}) || RunErl =:= false],
+    Erl = string:split(ct:get_progname()," ",all),
+    RunErlDir = filename:join(proplists:get_value(priv_dir, Config),?FUNCTION_NAME),
+    filelib:ensure_path(RunErlDir),
+    {ok, Peer, _Node} =
+        peer:start(
+          #{ name => ?CT_PEER_NAME(),
+             exec => {RunErl, Erl},
+             detached => false,
```

Comment:
Could you add a test to veify that `detached => false` works with both TCP and `standard_io` alternative connections? Technically I do that by adding test case name to `alternative()` function so it gets added to two test groups.

If it is not intended to, it should be disallowed in `verify_args/1` function with relevant user-friendly error.

Reply:
Added tests (and functionality) for tcp connection type. Also expanded verify_args to disallow standard_io.